client/src/components/TrackList.vue

```diff
@@ -141,6 +142,35 @@ export default defineComponent({
     watch(selectedTrackIdRef, scrollToTrack);
     watch(tracksRef, scrollToSelectedTrack);
 
+
+    async function multiDelete() {
+      const tracksDisplayed: number[] = [];
+      const text = ['Do you want to delete the following tracks:'];
+      let count = 0;
+      const limit = 20;
+      virtualListItems.value.forEach((item) => {
+        if (item.checkedTrackIds.includes(item.track.trackId)) {
+          if (count < limit) {
+            tracksDisplayed.push(item.track.trackId);
+            text.push(item.track.trackId.toString());
+          }
+          count += 1;
+        }
+      });
+      if (count >= limit) {
+        text.push(`And ${count - limit} more tracks...`);
+      }
+
+      const result = await prompt({
+        title: 'Confirm',
+        text,
+        confirm: true,
+      });
+      if (result) {
+        removeTrack(tracksDisplayed);
+      }
+    }
```

Comment:
I don't think this is working as intended.  If I have a few hundred tracks and I delete them, it lists 20 and says "...and 300 more", but then it only deletes the 20.

Clearly only `count < limit` tracks are getting added to `tracksDisplayed`, but then they're the only ones getting deleted.